crawlers.go

```diff
@@ -1361,7 +1369,7 @@ var crawlers = []string{
 	`YahooMailProxy`,
 	`YahooYSMcm`,
 	`YandeG`,
-	`yandex\.com\/bots`,
+	`Yandex`,
```

Comment:
We can't block Yandex. It will block traffic from Yandex.Dzen

Reply:
got it